bigbluebutton-html5/public/locales/en.json

```diff
@@ -280,7 +280,8 @@
     "app.meeting.alertBreakoutEndsUnderMinutesPlural": "Breakout is closing in {0} minutes.",
     "app.meeting.alertBreakoutEndsUnderMinutesSingular": "Breakout is closing in one minute.",
     "app.presentation.hide": "Hide presentation",
-    "app.presentation.notificationLabel": "Current presentation",
+    "app.presentation.newPresentationActiveNotification": "A new presentation is now active: {0}",
```

Comment:
This label doesn't seem very accurate. This message can lead to the misinterpretation that this "new" current presentation has just been uploaded, which is not the case for when you simply change the current presentation amongst your list of presentations. Bearing this in mind, I'd suggest something like: "Current active presentation: {0}". See if you agree with me, and please share your thoughts on it.

Reply:
That makes sense. I updated to `The current presentation is now: {0}`